### ISSUE TITLE
EVIO to LCIO converter for Hodoscope data

### ISSUE THREAD

**Issue by JeremyMcCormick** [JeffersonLab/hps-java#434]
- bank structure same as ecal
- FADC mode 1
- currently slots 2 and 4 (will change for run)

**Comment by rafopar** [comment]
> * bank structure same as ecal
> * FADC mode 1
> * currently slots 2 and 4 (will change for run)

It is actually slots 3 and 4 for now, later it will be slot10

**Comment by rafopar** [comment]
@JeremyMcCormick 

I put three evio files into the directory `/work/hallb/hps/rafopar/iss434_Tests/`.

The file `evio2lcio_command.dat` shows the command that I use to convert the file:

```
java -cp hps-distribution-4.4-SNAPSHOT-bin.jar org.hps.evio.EvioToLcio -d HPS-HodoscopeTest-v1 -R 1000000 hpshodo_000322.evio.00004 -l testout.slcio
```

The jar file is also present in this directory.

The evio file `hpshodo_000322.evio.00004` has all channels from both slot 3 and slot 4 data.  In the EVIO file `hpshodo_000350.evio.00000` channels 2, 3 and 9 are masked, so no data from these channels.

In the file `hpshodo_000351.evio.00000` in addition to run 350, all channels from slot 4 are also masked.

**Comment by JeremyMcCormick** [comment]
This is resolved by PR #467.
